### PR TITLE
add Chinese distill-whisper fine-tuning results 

### PR DESCRIPTION
**Pull request by yuekaizhang**
In #1605, we fine-tuned whisper using 14k hours Chinese data. This PR added decoding results for distill-whisper fine-tuning experiment.

Instead of actually using distillation loss for training, the model structure and parameter initialization method from the distill-whisper paper (https://arxiv.org/abs/2311.00430) were adopted: only the first and last layers of the decoder were retained.

Accuracy:
Distill-whisper is slightly worse comparing with norm whisper.   
| Model  | CER (Average SPEECH_IO_TEST_SET 01-26) | Training Set |
| --- | --- | --- |
| **whisper-large-ft-v1** | **4.32%** | multi-hans-zh (about 14k hours) |
| **whisper-large-ft-v1-distill** | **4.71%** | multi-hans-zh (about 14k hours) |

Speed: 
Every decoding step could acclerate about 4x comparing with the original decoder.
Norm whisper: 32 decoder layers
image
Distill-whisper: 2 decoder layers
image

For a quick test: https://huggingface.co/yuekai/icefall_asr_multi-hans-zh_whisper/blob/main/test_model.py

### CODE REVIEW

**Review by JinZr — APPROVED** [review]
LGTM, thanks!
